scripts/test/lint.py

```diff
@@ -86,6 +86,26 @@ def run_cmd(cmd):
     return out
 
 
+class IncludesDescriptionLength(Lint):
```

Comment:
The name is confusing, what is exactly included?

```suggestion
class InvalidDescriptionLength(Lint):
```